boolean/nand/a5f368aa-2cea-4110-a170-c08ec18a6a63.json

```diff
@@ -4,18 +4,21 @@
   "interface": {
     "inputs": {
       "ad9e2497-ee64-46bc-ae85-86ce71a33bd9": {
-        "type": "boolean\n",
-        "name": "a"
+        "type": "boolean",
+        "name": "a",
+        "description": "Must be a boolean eg true or false"
       },
       "be8c4674-102e-4f03-a086-5ed02f3c41b6": {
         "type": "boolean",
-        "name": "b"
+        "name": "b",
+        "description": "Must be a boolean eg true or false"
```

Comment:
For coherence with the other boolean nodes (like `boolean/And` I suggest this simple description:
"Receives the second operand"